README.md

```diff
@@ -15,6 +15,12 @@ Enables Ruff to be used in any editor that supports the LSP, including [Neovim](
 
 For Visual Studio Code, check out the [Ruff VS Code extension](https://github.com/astral-sh/ruff-vscode).
 
+## Using as a Python LSP
+
+ruff-lsp only supports surfacing errors and providing actions to correct them, it does not support behaviors for
+code navigation or completion. It's recommended to use ruff-lsp alongside another Python LSP in order to get
+a complete experience including features like "go-to definition" or auto-complete.
```

Comment:
I'd suggest removing the header so that `Highlights` and `Installation` remove the most prominent section.

What do you think of this, to make it a little more succinct?

> `ruff-lsp` supports surfacing Ruff diagnostics and providing Code Actions to fix them, but is intended to be used alongside another Python LSP in order to support features like navigation and autocompletion.

Reply:
I like that change.

I want to be able to link to a header though.. hm.

Reply:
Could we _also_ put this in an FAQ at the bottom that we can link to?

Reply:
I don't really like FAQs and wanted to keep this in the top so it's seen often. Let's just leave out the extra header for now and duplicate it in an FAQ if it gets annoying.